cmake/AddBipedalLocomotionYARPDevice.cmake

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+function(add_bipedal_yarp_device)
+  set(options )
+  set(oneValueArgs NAME)
+  set(multiValueArgs
+    SOURCES
+    PUBLIC_HEADERS
+    PRIVATE_HEADERS
+    PUBLIC_LINK_LIBRARIES
+    PRIVATE_LINK_LIBRARIES
+    DEFAULT
```

Comment:
I would argue that ``DEFAULT`` is a little hard to guess what it is needed for. As far as I understood, it is used to have the plugin compiled by default or not. Since we have already our logic for that, either we remove this input parameter, or we give it a better name.

Reply:
I removed it in https://github.com/dic-iit/bipedal-locomotion-framework/pull/130/commits/553b80ad0595da3d0346c3112f718c21d1e85f59